documentation/style-guide.md

```diff
@@ -0,0 +1,179 @@
+- For [General](https://www.codecademy.com/resources/docs/general) concepts, the entry can be any computer/code related subject not broad enough for its own topic entry.
+- Concepts typically have at least a few paragraphs and multiple sections. (see: [Headers/Sections](#headerssections))
+- We provide a [template for a concept entry](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md).
+### Terms
+- Are provided as a list under each parent [concept](#concepts).
+- Cover small elements of a concept, typically a method, function, property, or command.
+- Terms are typically shorter in length than concepts.
+- Terms typically have no more than two or three sections. (see: [Headers/Sections](#headerssections))
+- We provide a [template for a term entry](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md).
+
+## Writing Entries
+
+- Entries _must_ be completely original work. Copying from other websites (in whole or in part) is plagiarism and is unacceptable in Docs.
+- Entries are all written using Markdown.
+- Do not use HTML in place of Markdown formatting.
+- Bold the first occurrence of the concept or term in the body of the entry.
+- Bold should not be used for emphasis.
+- Use italics for emphasis when necessary.
+- Do not use italics for new terms.
+- Always expand and explain abbreviations when using them for the first time.
+- When referring to properties and methods, include the leading dot, and for methods include trailing parenthesis: `.property`, `.method()`.
+- When referring to elements as they would appear in code (properties, methods, functions, commands, expressions, etc.) use Markdown’s inline-code syntax i.e. backticks: (\`...\`). (see: [Code](#code))
+- With bulleted lists:
+  - A line break should precede and follow the list.
+  - There should be no line breaks between items in the list.
+  - Full sentences should be punctuated appropriately, with leading capitals and ending punctuation.
+- Avoid using first and second person pronouns, i.e. "I", "we", and "you".
+- Avoid words or phrases that assume the reader’s knowledge such as "easy", "obviously", "simply", "of course", and so on.
+- Always use an [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/).
+- Limit the entry to the concept or term being illustrated. i.e. if you find yourself writing about a method or property in a concept entry, it would be better to give it its own term entry.
+- Use a "note" to inform the reader of points of potential confusion or warn of potential pitfalls.
+  - A "note" occupies its own line with line breaks before and after.
+  - A "note" uses the Markdown blockquote format followed by the word "Note:" in bold, followed by the note text: `> **Note:** Note Text.`
```

Comment:
It might be a good idea to put in parenthesis what part represents the blockquote and what represents the bold.
```suggestion
  - A "note" uses the Markdown blockquote (`>`) format followed by the word "Note:" in bold (`**`), followed by the note text: `> **Note:** Note Text.`
```